sdk/playwrighttesting/microsoft-playwright-testing/src/core/playwrightService.ts

```diff
@@ -104,6 +105,7 @@ const getServiceConfig = (
         timeout: playwrightServiceConfig.timeout,
         exposeNetwork: playwrightServiceConfig.exposeNetwork,
         slowMo: playwrightServiceConfig.slowMo,
+        runName: playwrightServiceConfig.runName,
```

Comment:
why is this required here? runName is not a valid parameter for playwright connection options.